spring-boot-project/spring-boot/src/main/java/org/springframework/boot/info/GitProperties.java

```diff
@@ -126,4 +132,14 @@ private static Long parseEpochSecond(String s) {
 		}
 	}
 
+	static class GitPropertiesRuntimeHints implements RuntimeHintsRegistrar {
+
+		@Override
+		public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
+			// TODO MH: This path is configurable in ProjectInfoProperties!
```

Comment:
How to make this dynamic?

Reply:
Usually we register hints for the default location. This is the same for static resources & co.

Reply:
So there's no way to get the `Environment` or properties binding in the AOT phase, right? We just register the default locations and have to add something to the docs that users have to provide their own resource hints if they change that location?

Reply:
After a discussion with Stéphane we decided to just register the static locations, so this is fine.